extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -197,6 +212,17 @@ export class DataTierApplicationWizard {
 				page.dacFxPage.setupNavigationValidator();
 				page.dacFxPage.onPageEnter();
 			}
+
+			// Keeping track of the opened page, which is also useful for telemetry event for close wizard
+			this.currentPage = page === undefined ? 'Select an Operation page' : page.wizardPage.title;
```

Comment:
instead of creating a new variable to track this, it would be better to have a helper function called getCurrentPageTitle() to get the title. This would avoid potentially missing updating an additional variable somewhere.

Reply:
I tried if I can get the page title from any of the properties, but were accessible only during the page change, then added an  new variable to capture the current page. please let me now if i missed any thing to get the title. Thanks :)